src/actions.cr

```diff
@@ -7,14 +7,14 @@ module DppmRestApi::Actions
   extend self
   include RouteHelpers
 
+  ROUTE_MODULES    = [Pkg, App, Service, Src, User, Users, Groups]
   DEFAULT_DATA_DIR = "./data/"
   API_DOCUMENT     = DEFAULT_DATA_DIR + "api-options.json"
   alias ConfigKeyError = DPPM::Prefix::Base::ConfigKeyError
 
-  include Pkg
-  include App
-  include Service
-  include Src
+  {% for route_module in [Pkg, App, Service, Src, User, Users, Groups] %}
```

Comment:
It was perfectly good before, we have to write the module names anyway

Reply:
TBH I forgot I left this here when I pushed this morning... I should've known you wouldn't like it :p

I *always* do this sort of thing in personal projects. Have to type the same thing more than 3 or 4 times? Nope, that goes in a macro. Haha. I'll change it back when I go to make some changes on this next.

Reply:
I agree on the concept, but here in thia very case thats only `include` to add – doesn't worth the cost of macros IMO. There are even almost no characters saved, if it was the main point (at least for now)

Reply:
I agree on the concept, but here in thia's very case thats only `include` to add – doesn't worth the cost of macros IMO. There are even almost no characters saved, if it was the main point (at least for now)

Reply:
I agree on the concept, but here in this very case thats only `include` to add – doesn't worth the cost of macros IMO. There are even almost no characters saved, if it was the main point (at least for now). There are still hardcoded somehow.